app/models/reports/region_summary_schema.rb

```diff
@@ -245,6 +235,15 @@ def appts_scheduled_rates(entry)
       })
     end
 
+    def visit_details_rates(entry, with_ltfu: false)
```

Comment:
I named the aggregate of missed_visit, controlled, uncontrolled, visit_no_bp as `visit_details`.
Is there a better name to it?